etl/steps/data/garden/antibiotics/2024-10-23/animuse_trends.meta.yml

```diff
@@ -0,0 +1,23 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Global Health
+        - Antibiotics
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 365
+
+
+tables:
+  animuse_trends:
+    variables:
+      kilograms:
+        title: Quantity of antimicrobials used in animals
+        description_short: Reported quantity of antimicrobials used in animals.
```

Comment:
maybe a DoD would be useful and also you should probably either use _antibiotics_ (if that's the case) or _antimicrobials_ for all of your charts? Unless of course I am missing something and you've imported antibiotics for some of these datasets and antimicrobials for others

Reply:
Ah yes, it's all antimicrobials, but the dataset is largely made up of antibiotics. I think some countries don't report exactly what they've used, and some data is confidential, so I'll stick with using antimicrobials as I think that's the most accurate option here.